### PR TITLE
Changed build and deploy to deploy es site on pr too

### LINKED ISSUE #1016: ES site not deployed when doing PR?
lacking the _site_es directory on pre doesn't make the build fail, as it is never deployed when doing a PR. Probably the scripts need a proper cleaning.

### PR DESCRIPTION
**Pull request by alfredodev**
Resolves #1016

### CODE REVIEW

**Comment by alfredodev** [comment]
Resolves #1016

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
           at: .
       - run: |
           echo 'export FOLDER_NAME=site-${CIRCLE_BRANCH}' >> $BASH_ENV
-          echo 'export PREVIEW_URL=http://${BUCKET_NAME}.s3-website-eu-west-1.amazonaws.com/${FOLDER_NAME}/${WEBSITE_INDEX_DOCUMENT}' >> $BASH_ENV
+          echo 'export PREVIEW_URL=http://${BUCKET_NAME}.s3-website-eu-west-1.amazonaws.com/${FOLDER_NAME}/en/${WEBSITE_INDEX_DOCUMENT}' >> $BASH_ENV
```
Comment (Mattsi-Jansky) [inline .circleci/config.yml:41]: This is under the `production ` element, should it not be under `pullrequest`?

Comment (alfredodev) [reply .circleci/config.yml:41]: This PR is dead, too much has changed since.
But I'll take a look when we approach #1016 again, thanks!

Comment (MiyamotoAkira) [reply .circleci/config.yml:41]: If so, better cancel it.

**Pull request closed without merging**